app/templates/components/chat-message.hbs

```diff
@@ -1,5 +1,7 @@
-<time class="dt-published" datetime="">
-  [<a class="u-url time">{{time}}</a>]
-</time>
-<span class="from {{nickColorClass}}">{{message.from}}</span>
-<span class="text">{{formattedText}}</span>
+<div {{bind-attr class="isCurrentTimestamp:current"}}>
```

Comment:
`bind-attr` has been deprecated for quite a while. Should be `class="{{if iscurrentTimestamp 'current'}}"` instead.